thirdparty/cmdconfig/commands/cmdeval/cmdeval.go

```diff
@@ -431,7 +462,32 @@ func (r *EvalFnRunner) preRunE(c *cobra.Command, args []string) error {
 		// are deleted.
 		ContinueOnEmptyResult: true,
 		Selector:              r.Selector,
+		Exclusion:             r.Exclusion,
 	}
 
 	return nil
 }
+
+// parses annotation and label based selectors and exclusion from the command line input
+func (r *EvalFnRunner) parseSelectors() {
+	for _, s := range r.selectorAnno {
+		parts := strings.Split(s, "=")
+		key, value := parts[0], parts[1]
+		r.Selector.Annotations = map[string]string{key: value}
```

Comment:
just realized this is buggy (should be r.Selector.Annotations[key] = value). Updating now